subsys/net/lib/sockets/sockets.c

```diff
@@ -999,6 +1024,15 @@ int z_impl_zsock_poll(struct zsock_pollfd *fds, int nfds, int timeout)
 				continue;
 			}
 
+			/* The EINPROGRESS is returned if POLLOUT is done
+			 * to indicate that we can write to the socket
+			 * immediately.
+			 */
+			if (errno == EINPROGRESS) {
```

Comment:
Lol, so you didn't look at few lines immediately above, and just flopped a new code doing absolutely the same. Golden! So you see why I'm skeptical-by-default about such changes? Nobody gives a damn about existing code. Reading code is for cowards. Patch, patch, patch, patch!

Reply:
> Lol, so you didn't look at few lines immediately above, and just flopped a new code doing absolutely the same. Golden! So you see why I'm skeptical-by-default about such changes? Nobody gives a damn about existing code. Reading code is for cowards. Patch, patch, patch, patch!

I would appreciate that you stop making these kind of comments. These are not professional and productive.

Reply:
Do you hint that writing such code is professional and productive?